nsv13/code/modules/munitions/munitions.dm

```diff
@@ -361,9 +371,22 @@
 	chambered = null
 	loaded = null
 	firing = FALSE
+	maint_req = maint_req --
```

Comment:
Use defines

Reply:
define an integer?

Reply:
Yes, magic numbers bad.
Use something like
```#define MAINT_UNSCREWED 1
#define MAINT_UNWRENCHED 2```

Reply:
maint_req is still an integer guys, it a random number that counts down.

Reply:
yes but it's very unclear what state the integer represents if its simply a number. This is what enums are for in other languages but byond has nothing similar to that, which is where defines come in.

Reply:
So read the comment next to the var/ ?

Reply:
> yes but it's very unclear what state the integer represents if its simply a number. This is what enums are for in other languages but byond has nothing similar to that, which is where defines come in.

Well, in theory it has to do with the weapons system maintenance cycle given the notes, thus one could easily assume that it may be how many times you can use the thing before the thing needs to be maintained. We are all, in theory, smart enough to extrapolate from incomplete data about the appropriate unit and definition for that number.

Reply:
resolved: this is an variable and doesn't need to be defined, comments updated